### PR TITLE
Added getting started guide

### PR DESCRIPTION
**Pull request by chipsnyder**
@syellows @MichaelCampbell @tsivilay @hkellaway @OatLiew @zgeorge 

Created a getting started guide for new developers. Versions and tools listed below are the requirements set on the Jenkins machine and my machine. We'll want to make sure we're constant across all machines so make sure you check to see you're up to date. 

- [x] add missing account word https://github.com/AbercrombieAndFitch/Objective-C-Style-Guide/pull/4#discussion_r36093016
- [x] Link to bundler site https://github.com/AbercrombieAndFitch/Objective-C-Style-Guide/pull/4#discussion_r36093095

### CODE REVIEW

**Comment by hkellaway** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -37,6 +37,43 @@ Here are some of the documents from Apple that informed the style guide. If some
...
+* Installing:
+  * XCode can be installed from the [App Store](https://itunes.apple.com/us/app/xcode/id497799835?ls=1&mt=12)
+
+### OSX Yosemite
+Version: 10.10.3
+
+The latest version of Yosemite can be installed from the [App Store](https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=915041082&mt=12&ls=1)
+
+### GitHub
+If you haven't already please create a [GitHub](https://github.com/join) and notify your A&F team lead so they can get you added to the A&F private repos.
```
Comment (MichaelCampbell) [inline README.md]: > If you haven't already please create a [GitHub](https://github.com/join) and

I think you're missing the word ***account***

File: README.md [hunk 2]
```diff
@@ -37,6 +37,43 @@ Here are some of the documents from Apple that informed the style guide. If some
...
+
+### Project Specific tools
+Each project manages it's own `gem` and `pod` dependencies. RubyGems (including the CocoaPod gem) are managed by Bundler (installed during the [RVM](#rvm) setup.) Each Project has a `Gemfile` and one or more `Podfile`s. To install that projects dependencies execute:
```
Comment (MichaelCampbell) [inline README.md]: I think we should also link to the Bundler site for convenience

**Comment by MichaelCampbell** [comment]
:+1: once the 2 tasks are addressed. Thanks for doing this @chipsnyder

**Pull request merged**